ompi/mca/pml/base/pml_base_frame.c

```diff
@@ -124,6 +139,20 @@ static int mca_pml_base_register(mca_base_register_flag_t flags)
     (void) mca_base_var_register_synonym(var_id, "ompi", "pml", NULL, "wrapper", 0);
 #endif
 
+#if MPI_VERSION >= 4
+    mca_base_var_enum_create("pml_base_deprecate_warnings", ompi_pml_base_deprecation_warnings,
+                             &new_enum);
+    rc = mca_base_var_register("ompi", "pml", "base", "warn_deprecated",
+                               "Specify number of deprecation warnings",
```

Comment:
```suggestion
                               "How often to issue warnings for deprecated features",
```

Reply:
```suggestion
                               "How often to issue warnings for deprecated features (e.g., canceling send operations)",
```
I wonder if we should make this "cancel send" specific. Since it is the only thing we warn about.

Reply:
I was wondering that too and couldn't make up my mind last night. There is the occasional push in the Forum to deprecate cancellation of receive requests but I don't think it has enough traction right now. I think being more specific here is good and we can always broaden the scope if we have to.